app/src/i18n/ui.ts

```diff
@@ -4,8 +4,6 @@ import es from './locales/es/_export'
 import jp from './locales/jp/_export'
 import ko from './locales/ko/_export'
 
-/* Changing the languages here will automatically update astro config */
```

Comment:
I think it's good to leave this comment for any future developer to know this when reading the file.